app/swagger/util.js

```diff
@@ -0,0 +1,45 @@
+exports.deviceIdHeader = {
+    deviceId: {
+        type: 'string',
+        description: 'This is Device Id of user!'
+    }
+}
+
+exports.jwtTokenHeader = {
+    authorization: {
+        type: 'string',
+        description: 'This is JWT token!'
+    }
+}
+
+exports.responseError = (errorCodes) => {
+    const errorResponses = {};
+    errorCodes.forEach(errorCode => {
+        errorResponses[errorCode] = {
+            code: {
+                type: 'number', 
+                example: errorCode
+            },
+            errorMessage: {
+                type: 'string', 
+                example: getErrorMessage(errorCode)
+            }
+        }
+    });
+
+    return errorResponses;
+}
+
+function getErrorMessage(errorCode) {
+    switch(errorCode) {
+        case 400: return 'Missing token! OR Invalid user!'
+        case 401: return 'Invalid token! OR User not found!'
+        case 403: return 'User is not yet approved!'
+        case 422: return 'User not logged in through own device! OR Invalid request!'
+        case 409: return 'User exist!'
+        case 504: return 'Downstream service error'
+        case 502: return 'Failed to add user! OR Failed to reset/forgot user\'s password OR update/approve User\'s profile!'
+        case 500: return 'Authorization error OR Something went wrong!'
```

Comment:
❓ Why are you putting a general message for these error codes? Can't swagger itself look for the error message from responses?

Reply:
Hey currently have not linked Swagger with apis so you will not be able to try on real time data (purpose of hiding is currently our apis are public so anyone will be able to play around with it so once they are private then will enable Swagger Try It option).

Also, created this util for displaying description in the form of samples. So, this will tell you what kind of errors you can expect from this api.

We are planning to use swagger as api documentation and for real time api calls please use postman. (I have shared postman apis json file and shared steps on how to use it)